modules/db-writer/src/main/resources/analytics.sql

```diff
@@ -7,7 +7,7 @@ drop table executed_deposit;
 drop table executed_redeem;
 drop table executed_swap;
 
-create table if not exists executed_deposit (
+create table if not exists deposit (
```

Comment:
in every order there is requested tokens amount and the real one

Reply:
Yep. Current version of table schemas is legacy. So, will be changed in next pr